modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/release-notes-features-template/page-definition.json

```
@@ -43,7 +43,7 @@
 								"align": "Start",
 								"contentDisplay": "FlexRow",
 								"justify": "SpaceBetween",
-								"marginBottom": 1,
+								"marginBottom": 1.5,
```

update for this, as an error is occurring when creating the Display Page Template.

```suggestion
								"marginBottom": 2,
```